Install headers, libraries and programs with `cp -a` so their timestamps are kept:
a reinstall that changes nothing then does not look newer to every dependent port's make.

--- a/Configurations/unix-Makefile.tmpl	2026-09-30 12:40:03.082112483 +0200
+++ b/Configurations/unix-Makefile.tmpl	2026-09-30 12:40:03.088604226 +0200
@@ -682,7 +682,7 @@
 	@$(PERL) $(SRCDIR)/util/mkdir-p.pl "$(DESTDIR)$(OPENSSLDIR)"
 	@$(ECHO) "*** Installing FIPS module"
 	@$(ECHO) "install $(INSTALL_FIPSMODULE) -> $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME)"
-	@cp "$(INSTALL_FIPSMODULE)" "$(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).new"
+	@cp -a "$(INSTALL_FIPSMODULE)" "$(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).new"
 	@chmod 755 "$(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).new"
 	@mv -f "$(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).new" \
 	       "$(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME)"
@@ -755,21 +755,21 @@
 	@$(PERL) $(SRCDIR)/util/mkdir-p.pl "$(DESTDIR)$(INSTALLTOP)/include/openssl"
 	@ : {- output_off() if $disabled{uplink}; "" -}
 	@$(ECHO) "install $(SRCDIR)/ms/applink.c -> $(DESTDIR)$(INSTALLTOP)/include/openssl/applink.c"
-	@cp $(SRCDIR)/ms/applink.c "$(DESTDIR)$(INSTALLTOP)/include/openssl/applink.c"
+	@cp -a $(SRCDIR)/ms/applink.c "$(DESTDIR)$(INSTALLTOP)/include/openssl/applink.c"
 	@chmod 644 "$(DESTDIR)$(INSTALLTOP)/include/openssl/applink.c"
 	@ : {- output_on() if $disabled{uplink}; "" -}
 	@set -e; for i in $(SRCDIR)/include/openssl/*.h \
 			  $(BLDDIR)/include/openssl/*.h; do \
 		fn=`basename $$i`; \
 		$(ECHO) "install $$i -> $(DESTDIR)$(INSTALLTOP)/include/openssl/$$fn"; \
-		cp $$i "$(DESTDIR)$(INSTALLTOP)/include/openssl/$$fn"; \
+		cp -a $$i "$(DESTDIR)$(INSTALLTOP)/include/openssl/$$fn"; \
 		chmod 644 "$(DESTDIR)$(INSTALLTOP)/include/openssl/$$fn"; \
 	done
 	@$(PERL) $(SRCDIR)/util/mkdir-p.pl "$(DESTDIR)$(libdir)"
 	@set -e; for l in $(INSTALL_LIBS); do \
 		fn=`basename $$l`; \
 		$(ECHO) "install $$l -> $(DESTDIR)$(libdir)/$$fn"; \
-		cp $$l "$(DESTDIR)$(libdir)/$$fn.new"; \
+		cp -a $$l "$(DESTDIR)$(libdir)/$$fn.new"; \
 		$(RANLIB) "$(DESTDIR)$(libdir)/$$fn.new"; \
 		chmod 644 "$(DESTDIR)$(libdir)/$$fn.new"; \
 		mv -f "$(DESTDIR)$(libdir)/$$fn.new" \
@@ -791,7 +791,7 @@
 		: {- output_off() unless windowsdll() or sharedaix() or sharedaix_solib(); output_on() if windowsdll() or sharedaix_solib(); "" -}; \
 		if [ "$$fn3" != "" ]; then \
 			$(ECHO) "install $$s3 -> $(DESTDIR)$(libdir)/$$fn3"; \
-			cp $$s3 "$(DESTDIR)$(libdir)/$$fn3.new"; \
+			cp -a $$s3 "$(DESTDIR)$(libdir)/$$fn3.new"; \
 			chmod 755 "$(DESTDIR)$(libdir)/$$fn3.new"; \
 			mv -f "$(DESTDIR)$(libdir)/$$fn3.new" \
 			      "$(DESTDIR)$(libdir)/$$fn3"; \
@@ -801,7 +801,7 @@
 		$(ECHO) "install $$s1 -> $$a"; \
 		if [ -f $$a ]; then ( trap "rm -rf /tmp/ar.$$$$" INT 0; \
 			mkdir /tmp/ar.$$$$; ( cd /tmp/ar.$$$$; \
-			cp -f $$a $$a.new; \
+			cp -a -f $$a $$a.new; \
 			for so in `$(AR) t $$a`; do \
 				$(AR) x $$a $$so; \
 				chmod u+w $$so; \
@@ -818,14 +818,14 @@
 	@for e in $(INSTALL_EXPORTERS_PKGCONFIG); do \
 		fn=`basename $$e`; \
 		$(ECHO) "install $$e -> $(DESTDIR)$(PKGCONFIGDIR)/$$fn"; \
-		cp $$e "$(DESTDIR)$(PKGCONFIGDIR)/$$fn"; \
+		cp -a $$e "$(DESTDIR)$(PKGCONFIGDIR)/$$fn"; \
 		chmod 644 "$(DESTDIR)$(PKGCONFIGDIR)/$$fn"; \
 	done
 	@$(PERL) $(SRCDIR)/util/mkdir-p.pl "$(DESTDIR)$(CMAKECONFIGDIR)"
 	@for e in $(INSTALL_EXPORTERS_CMAKE); do \
 		fn=`basename $$e`; \
 		$(ECHO) "install $$e -> $(DESTDIR)$(CMAKECONFIGDIR)/$$fn"; \
-		cp $$e "$(DESTDIR)$(CMAKECONFIGDIR)/$$fn"; \
+		cp -a $$e "$(DESTDIR)$(CMAKECONFIGDIR)/$$fn"; \
 		chmod 644 "$(DESTDIR)$(CMAKECONFIGDIR)/$$fn"; \
 	done
 
@@ -894,7 +894,7 @@
 		if [ "$$e" = "dummy" ]; then continue; fi; \
 		fn=`basename $$e`; \
 		$(ECHO) "install $$e -> $(DESTDIR)$(ENGINESDIR)/$$fn"; \
-		cp $$e "$(DESTDIR)$(ENGINESDIR)/$$fn.new"; \
+		cp -a $$e "$(DESTDIR)$(ENGINESDIR)/$$fn.new"; \
 		chmod 755 "$(DESTDIR)$(ENGINESDIR)/$$fn.new"; \
 		mv -f "$(DESTDIR)$(ENGINESDIR)/$$fn.new" \
 		      "$(DESTDIR)$(ENGINESDIR)/$$fn"; \
@@ -918,7 +918,7 @@
 		if [ "$$e" = "dummy" ]; then continue; fi; \
 		fn=`basename $$e`; \
 		$(ECHO) "install $$e -> $(DESTDIR)$(MODULESDIR)/$$fn"; \
-		cp $$e "$(DESTDIR)$(MODULESDIR)/$$fn.new"; \
+		cp -a $$e "$(DESTDIR)$(MODULESDIR)/$$fn.new"; \
 		chmod 755 "$(DESTDIR)$(MODULESDIR)/$$fn.new"; \
 		mv -f "$(DESTDIR)$(MODULESDIR)/$$fn.new" \
 		      "$(DESTDIR)$(MODULESDIR)/$$fn"; \
@@ -949,13 +949,13 @@
 		fn=`basename $$s`; \
 		: {- output_off() unless windowsdll(); "" -}; \
 		$(ECHO) "install $$s -> $(DESTDIR)$(bindir)/$$fn"; \
-		cp $$s "$(DESTDIR)$(bindir)/$$fn.new"; \
+		cp -a $$s "$(DESTDIR)$(bindir)/$$fn.new"; \
 		chmod 755 "$(DESTDIR)$(bindir)/$$fn.new"; \
 		mv -f "$(DESTDIR)$(bindir)/$$fn.new" \
 		      "$(DESTDIR)$(bindir)/$$fn"; \
 		: {- output_on() unless windowsdll(); "" -}{- output_off() if windowsdll(); "" -}; \
 		$(ECHO) "install $$s -> $(DESTDIR)$(libdir)/$$fn"; \
-		cp $$s "$(DESTDIR)$(libdir)/$$fn.new"; \
+		cp -a $$s "$(DESTDIR)$(libdir)/$$fn.new"; \
 		chmod 755 "$(DESTDIR)$(libdir)/$$fn.new"; \
 		mv -f "$(DESTDIR)$(libdir)/$$fn.new" \
 		      "$(DESTDIR)$(libdir)/$$fn"; \
@@ -970,7 +970,7 @@
 		if [ "$$x" = "dummy" ]; then continue; fi; \
 		fn=`basename $$x`; \
 		$(ECHO) "install $$x -> $(DESTDIR)$(bindir)/$$fn"; \
-		cp $$x "$(DESTDIR)$(bindir)/$$fn.new"; \
+		cp -a $$x "$(DESTDIR)$(bindir)/$$fn.new"; \
 		chmod 755 "$(DESTDIR)$(bindir)/$$fn.new"; \
 		mv -f "$(DESTDIR)$(bindir)/$$fn.new" \
 		      "$(DESTDIR)$(bindir)/$$fn"; \
@@ -979,7 +979,7 @@
 		if [ "$$x" = "dummy" ]; then continue; fi; \
 		fn=`basename $$x`; \
 		$(ECHO) "install $$x -> $(DESTDIR)$(bindir)/$$fn"; \
-		cp $$x "$(DESTDIR)$(bindir)/$$fn.new"; \
+		cp -a $$x "$(DESTDIR)$(bindir)/$$fn.new"; \
 		chmod 755 "$(DESTDIR)$(bindir)/$$fn.new"; \
 		mv -f "$(DESTDIR)$(bindir)/$$fn.new" \
 		      "$(DESTDIR)$(bindir)/$$fn"; \
